books/microbiology/bake

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '5.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :microbiology) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+  metadata = book.metadata
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+
+  BakePreface.v1(book: book, title_element: 'h1')
+  BakeChapterIntroductions.v1(book: book)
+  BakeChapterTitle.v1(book: book)
+  book.chapters.each do |chapter|
+    BakePageAbstracts.v1(chapter: chapter)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+  end
+
+  BakeAutotitledNotes.v1(
+    book: book,
+    classes: %w[clinical-focus check-your-understanding micro-connection link-to-learning eye-on-ethics case-in-point disease-profile]
+  )
+
+  book_answer_key = BookAnswerKeyContainer.v1(book: book, solutions_plural: false)
+  book.chapters.each do |chapter|
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata)
+
+    eoc_wrapper = ChapterReviewContainer.v1(chapter: chapter, metadata_source: metadata, klass: 'review-questions')
+
+    # EoC sections with exercises
+    classnames = %w[multiple-choice true-false matching fill-in-the-blank short-answer critical-thinking]
+
+    classnames.each do |klass|
+      MoveExercisesToEOC.v1(chapter: chapter, metadata_source: metadata, append_to: eoc_wrapper, klass: klass)
+    end
+
+    chapter.search('.multiple-choice, .true-false, .matching, .fill-in-the-blank, .short-answer, .critical-thinking').exercises.each do |exercise|
```

Comment:
Seems a shame to not reuse the array of class names.  Plus this line is pretty long.

```ruby
chapter.search(exercise_section_classes.map{|klass| ".#{klass}"}.join(', '))
```

Also remember how much faster searches are when the element name is included, so may instead want

```ruby
chapter.search(exercise_section_classes.map{|klass| "div.#{klass}"}.join(', '))
```